rc/.eslintrc.json

```diff
@@ -22,14 +23,18 @@
         "no-cond-assign": 2,
         "no-console": 2,
         "no-constant-condition": 2,
+        "no-continue": 0,
```

Comment:
Too much code in one PR; let's break it out into multiple PRs

 - one PR for the --es6 flag
 - one PR for line-length 
 - one PR for all this other churn in this file.

Feel free to put --es6 and --line-length into a single PR if you want; those two look fine.

It's all this churn in this file that needs detailed review.